public/scripts/ticketTranscript.js

```
@@ -0,0 +1,557 @@
+  document.getElementById("transcriptContent").style.display = "none";
+  document.getElementById("errorState").style.display = "none";
+  document.getElementById("loadingState").style.display = "block";
+  loadTranscript();
+}
+
+function handleKeyboardShortcuts(e) {
+  if (e.ctrlKey || e.metaKey) {
+    switch (e.key) {
+      case "f":
+        e.preventDefault();
+        const searchTerm = prompt("Search in messages:");
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)